quickstart.yml

```
@@ -29,14 +29,14 @@ services:
       - URLS_LOGOUT=http://127.0.0.1:3000/logout
       - DSN=memory
       - SECRETS_SYSTEM=youReallyNeedToChangeThis
-      - OIDC_SUBJECT_TYPES_SUPPORTED=public,pairwise
-      - OIDC_SUBJECT_TYPE_PAIRWISE_SALT=youReallyNeedToChangeThis
+      - OIDC_SUBJECT_IDENTIFIERS_ENABLED=public,pairwise
```

The naming of this key was actually wrong, I've fixed the naming in #1706 . The proper key is now:

```suggestion
      - OIDC_SUBJECT_IDENTIFIERS_SUPPORTED_TYPES=public,pairwise
```

Config Key `OIDC_SUBJECT_IDENTIFIERS_ENABLED` is still valid but will show a deprecation warning.